src/lib/tasks/autoresearchTaskExecutor.ts

```
@@ -0,0 +1,329 @@
+              toolName: phasePolicy.forcedTool,
+            },
+          }
+        : {}),
+      // Observability
+      requestId: runId,
+      ...(task.metadata?.maxBudgetUsd
+        ? { maxBudgetUsd: task.metadata.maxBudgetUsd }
+        : {}),
+    };
+
+    let result;
```

Biome (2.4.10)

[error] This variable implicitly has the any type.

(lint/suspicious/noImplicitAnyLet)